src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export function getVisibleMovies(
+  movies: Movie[],
+  query: string,
+) {
+  const normalizedQuery = query.toLowerCase();
+
+  return movies.filter(movie => movie.title
+    .toLowerCase().includes(normalizedQuery)
+    || movie.description.toLowerCase().includes(normalizedQuery));
```

Comment:
return movies.filter(movie => (
     movie.title.toLowerCase().includes(normalizedQuery)
    || movie.description.toLowerCase().includes(normalizedQuery)));

in my opinion, its more readable

Reply:
Agreed. Changed it to your variant.